.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.formatOnSave": true,
+  "prettier.eslintIntegration": true
+}
```

Comment:
this should not be committed. Please add this file to `.gitignore`

Reply:
Ooops